src/main/java/core/basesyntax/service/impl/activity/ActivityHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service.impl.activity;
```

Comment:
the same about package ```activity```  it should  be subpackage ```strategy``` and you can name it ```impl``` because  it is the implementation of your strategy